### PR TITLE
Initial P2P

### PR DESCRIPTION
**Pull request by tmcalves**

### CODE REVIEW

File: .github/workflows/execute-command.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        required: true
+        type: string
+      dry-run:
+        required: false
+        type: boolean
+        default: false
+      zone:
+        required: false
+        type: string
+        default: europe-west2
```
Comment (chbatey) [inline .github/workflows/execute-command.yaml:25]: zone and region are the wrong way around

File: .github/workflows/execute-command.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      region:
+        required: false
+        type: string
+        default: europe-west2-a
+env:
+  REGISTRY: europe-west2-docker.pkg.dev/${{ inputs.project-id }}/tenant
```
Comment (chbatey) [inline .github/workflows/execute-command.yaml:31]: this should use the region var

File: .github/workflows/increment-version.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+
+name: p2p-command
```
Comment (chbatey) [inline .github/workflows/increment-version.yaml:2]: diff name?

File: .github/workflows/p2p.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
+name: P2P
```
Comment (chbatey) [inline .github/workflows/p2p.yaml:1]: Should we call this, and its file name idp-p2p? I am just thinking about future proofing us providing a CECG P2P that doesn't do any IDP stuff e.g. the login etc

File: README.md [hunk 5]
```diff
@@ -1,2 +1,55 @@
 # reusable-p2p
```
Comment (chbatey) [inline README.md:1]: Let's also mention in the read me this is specifically for the CECG IDP, as it does the login etc

**Review by chbatey — COMMENTED** [review]
Looking good, a few comments

File: README.md (hunk 5 above)
Comment (tmcalves) [reply README.md]: Done!

**Comment by tmcalves** [comment]
Addressed comments

**Review by chbatey — APPROVED** [review]
LGTM :)

**Pull request merged**